### PR TITLE
Add RDM Spell Frazzle to Mhaura NPC Tya Padolih

### PR DESCRIPTION
**Pull request by tristamgreen**
Adds Frazzle to the stock of NPC [Tya Padolih](http://ffxiclopedia.wikia.com/wiki/Tya_Padolih).

### CODE REVIEW

**Comment by takhlaq** [comment]
thanks!
in future, please use descriptive commit messages (update <file> tells us nothing about why it was updated)

**Comment by tristamgreen** [comment]
will do, thanks for letting me know :)

**Comment by pandorraffxi** [comment]
@takhlaq - safe to say if we are updating luas to convert hex > dec?

**Comment by takhlaq** [comment]
yeah i'll run a mass replace some time

File: scripts/zones/Mhaura/npcs/Tya_Padolih.lua [hunk 1]
```diff
@@ -33,7 +33,8 @@ stock = {0x126c,4147,  --Scroll of Regen
          0x1288,5523,  --Scroll of Invisible
          0x1289,2400,  --Scroll of Sneak
          0x128a,1243,  --Scroll of Deodorize
-         0x1330,18032} --Scroll of Distract
+         0x1330,18032  --Scroll of Distract
```
Comment (Ninjistix) [inline scripts/zones/Mhaura/npcs/Tya_Padolih.lua]: missing comma after old list, 0x1330,18032,

Comment (TeoTwawki) [reply scripts/zones/Mhaura/npcs/Tya_Padolih.lua]: I'll get it

Comment (TeoTwawki) [reply scripts/zones/Mhaura/npcs/Tya_Padolih.lua]: #4678

**Pull request merged**